solus_sc/changelog.py

```diff
@@ -52,6 +61,8 @@ def decode_changelog(self, text):
         ]
         block_elem_ids = ["{}:".format(x) for x in block_elems]
 
+        text = MARKUP_CODE_BLOCK_HIT.sub(r'<tt>\n\2</tt>', text)
```

Comment:
```suggestion
        text = MARKUP_CODE_BLOCK_HIT.sub(r'<tt>\2</tt>', text)
```
Since code blocks are already separated by a line consisting of backticks this ends up creating too much whitespace otherwise (see e.g. the xpra 5.0.2 changelog)